lib/matplotlib/contour.py

```diff
@@ -1180,15 +1180,34 @@ def _autolev(self, N):
         one contour line, but two filled regions, and therefore
         three levels to provide boundaries for both regions.
         """
+        self._auto = True
         if self.locator is None:
             if self.logscale:
                 self.locator = ticker.LogLocator()
             else:
                 self.locator = ticker.MaxNLocator(N + 1, min_n_ticks=1)
 
         lev = self.locator.tick_values(self.zmin, self.zmax)
-        self._auto = True
-        return lev
+
+        try:
+            if self.locator._symmetric:
+                return lev
+        except AttributeError:
+            pass
+
+        under = np.nonzero(lev < self.zmin)[0]
```

Comment:
This code block could use a comment - I didn't find it easy to parse and would only have known because you explained it to me on the weekly call...

Reply:
Added.